addons/policies/bootstrap_relationship_policies.json

```diff
@@ -913,6 +913,86 @@
                     "remove-relationship"
                 ]
             }
+        },
+        {
+            "typeName": "AuthPolicy",
+            "attributes":
+            {
+                "name": "LINK_AI_MODEL_TO_AI_APPLICATION",
+                "qualifiedName": "LINK_AI_MODEL_TO_AI_APPLICATION",
+                "policyCategory": "bootstrap",
+                "policySubCategory": "default",
+                "policyServiceName": "atlas",
+                "policyType": "allow",
+                "policyPriority": 0,
+                "policyUsers":
+                [],
+                "policyGroups":
+                [],
+                "policyRoles":
+                [
+                    "$admin",
+                    "$api-token-default-access"
+                ],
+                "policyResourceCategory": "RELATIONSHIP",
+                "policyResources":
+                [
+                    "end-one-entity-classification:*",
+                    "end-two-entity-classification:*",
+                    "end-one-entity:*",
+                    "end-two-entity:*",
```

Comment:
What are these stars for?

Reply:
Okay, this denotes qualified name, so * should be good since there is check for type as well.